src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -49,9 +51,42 @@ public void actionPerformed(ActionEvent actionEvent) {
 			}
 		};
 
+		ActionListener selectTestFigureTriangleListener = new ActionListener() {
+			@Override
+			public void actionPerformed(ActionEvent actionEvent) {
+				FigureFactory.getFigure(FigureFactory.Figure.TRIANGLE).execute(DriverFeature.getDriverManager().getCurrentDriver());
+			}
+		};
+
+		ActionListener selectTestFigureRectangleListener = new ActionListener() {
+			@Override
+			public void actionPerformed(ActionEvent actionEvent) {
+				FigureFactory.getFigure(FigureFactory.Figure.RECTANGLE).execute(DriverFeature.getDriverManager().getCurrentDriver());
+			}
+		};
```

Comment:
It is good to keep a convention used in the original code (class instance instead of anonymous classes)